### PR TITLE
Add Pypi boilerplate files to publish to pypi server

### PR DESCRIPTION
**Pull request by mmthomas55**
@jessmchung @mymac80 @mortaliorchard 
Please review.  After merge, I can publish it to our Pypi Cloud server which is now persisted to S3, through Jenkins http://jenkins.theorchard.com:8080/view/pypi/job/publish-pypi-package/.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,19 @@ Logging
...
-OWS1 standard. Installation is minimal:
+OWS1 standard. This package is published to our internal Pypi server.
+Installation is minimal:
 
 ```bash
-pip install git+https://github.com/theorchard/python-owslogger.git@#egg=owslogger
+pip install -i https://pypi.theorchard.io/pypi/owslogger
+```
+
+Alternatively, put the following in your requrements file:
```
Comment (jessmchung) [inline README.md]: requirements (typo)

File: requirements-dev.txt [hunk 2]
```diff
@@ -1 +1 @@
-pytest==2.8.7
\ No newline at end of file
+pytest==2.8.7bumpversion==0.5.3
```
Comment (jessmchung) [inline requirements-dev.txt]: alphabetize please

**Comment by jessmchung** [comment]
Looks good - just minor changes. R~

**Comment by mmthomas55** [comment]
Fixed.

**Comment by jessmchung** [comment]
R+

**Comment by mymac80** [comment]
R+

**Pull request merged**